README.md

```diff
@@ -90,17 +90,14 @@ Kops also has time set aside every other week to offer help and guidance to the
 community. Kops maintainers have agreed to set aside time specifically dedicated
 to working with newcomers, helping with PRs, and discussing new features.
 
-We recommend letting us know **beforehand** if you plan on attending so we can
-have time to prepare for the call.
+Please feel free to reach out **beforehand** if you plan on attending!
 
-| Maintainer   | Schedule      |  URL |
```

Comment:
I made a comment that we are not we. We are contributors, but we are community. I don't want people to look at us and compare themselves to us. Inclusion ... I know we are awesome. I don't think we need to broadcast it. K8s core does not have it, we are not 3, we are many.

Prime time

Go team!!!!